### PR TITLE
feat: allow access to secrets manager

### PR DESCRIPTION
**Pull request by stemey**

### CODE REVIEW

File: src/cdk/utils/create-lambda-integration.ts [hunk 1]
```diff
@@ -50,6 +56,23 @@ export function createLambdaIntegration(
...
+    resources: [
+      'arn:aws:secretsmanager:' +
+        stack.region +
+        ':' +
+        stack.account +
+        ':secret:*',
+    ],
+  });
+
+  const secretsPolicy = new aws_iam.Policy(stack, 'secrets-getvalue-policy', {
```
Comment (clebert) [inline src/cdk/utils/create-lambda-integration.ts:70]: Please rename variable to `secretsManagerPolicy` and adapt to the project style of naming IDs e.g. 'SecretsManagerPolicy'

File: src/cdk/utils/create-lambda-integration.ts [hunk 2]
```diff
@@ -50,6 +56,23 @@ export function createLambdaIntegration(
     }
   );
 
+  const secretsManagerPolicy = new aws_iam.PolicyStatement({
```
Comment (clebert) [inline src/cdk/utils/create-lambda-integration.ts:59]: Please rename to `secretsManagerPolicyStatement` or use it inline in line no. 71

File: src/cdk/utils/create-lambda-integration.ts [hunk 3]
```diff
@@ -50,6 +56,23 @@ export function createLambdaIntegration(
...
+    actions: ['secretsmanager:GetSecretValue'],
+    resources: [
+      'arn:aws:secretsmanager:' +
```
Comment (clebert) [inline src/cdk/utils/create-lambda-integration.ts:62]: Please use a template literal instead of string concatenation

**Review by clebert — CHANGES_REQUESTED** [review]
Why is the onecmd.js file empty? And the package-lock.json should not change. Do you have the latest npm version installed?

**Review by stemey — COMMENTED** [review]
prerelease

File: src/cdk/utils/create-lambda-integration.ts (hunk 2 above)
Comment (stemey) [reply src/cdk/utils/create-lambda-integration.ts]: check again

File: src/cdk/utils/create-lambda-integration.ts (hunk 3 above)
Comment (stemey) [reply src/cdk/utils/create-lambda-integration.ts]: done

**Review by clebert — APPROVED** [review]

**Pull request merged**